### DISCUSSION TITLE
Container from ghcr.io not showing as package in repo

### DISCUSSION THREAD

**Discussion by yajo** [community/community#26341] [Actions]
<p>Look here: <a href="https://github.com/orgs/Tecnativa/packages/container/package/docker-socket-proxy" rel="noopener nofollow ugc">https://github.com/orgs/Tecnativa/packages/container/package/docker-socket-proxy</a></p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/4/a/4a47351712f7fdc60774e7fe946c806e35620a2b.png" data-download-href="/uploads/short-url/aB5TqgaDpDaVEpV8iLLl5wtsVDB.png?dl=1" title="imagen" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/973709/181097197-2ec6a063-d3fe-4db9-85f7-ecbbea6c9619.png" alt="imagen" data-base62-sha1="aB5TqgaDpDaVEpV8iLLl5wtsVDB" width="690" height="486" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/4/a/4a47351712f7fdc60774e7fe946c806e35620a2b_2_690x486.png, https://aws1.discourse-cdn.com/github/original/3X/4/a/4a47351712f7fdc60774e7fe946c806e35620a2b.png 1.5x, https://aws1.discourse-cdn.com/github/original/3X/4/a/4a47351712f7fdc60774e7fe946c806e35620a2b.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/4/a/4a47351712f7fdc60774e7fe946c806e35620a2b_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">imagen</span><span class="informations">1006×709 60 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>The package is public, and linked to <a href="https://github.com/Tecnativa/docker-socket-proxy" rel="noopener nofollow ugc">https://github.com/Tecnativa/docker-socket-proxy</a> but if you go there, there are no packages:</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/2/e/2e1b71c49fa2d034976380554f5549365ab4ce4f.png" data-download-href="/uploads/short-url/6zSLj0JH7aES8MhPwzLG4YtFlF5.png?dl=1" title="imagen" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/973709/181097200-d07843bb-978a-44f3-b2a7-d4625145f56b.png" alt="imagen" data-base62-sha1="6zSLj0JH7aES8MhPwzLG4YtFlF5" width="690" height="268" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/2/e/2e1b71c49fa2d034976380554f5549365ab4ce4f_2_690x268.png, https://aws1.discourse-cdn.com/github/optimized/3X/2/e/2e1b71c49fa2d034976380554f5549365ab4ce4f_2_1035x402.png 1.5x, https://aws1.discourse-cdn.com/github/original/3X/2/e/2e1b71c49fa2d034976380554f5549365ab4ce4f.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/2/e/2e1b71c49fa2d034976380554f5549365ab4ce4f_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">imagen</span><span class="informations">1318×512 76.4 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>If you click on “Packages”, it goes to <a href="https://github.com/orgs/Tecnativa/packages?repo_name=docker-socket-proxy" rel="noopener nofollow ugc">https://github.com/orgs/Tecnativa/packages?repo_name=docker-socket-proxy</a> and also is empty:</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/c/2/c21a2b402e6b5effa09d581f3a15ab0dbf231191.png" data-download-href="/uploads/short-url/rH6H9ltmwKmlypnzDtkXr33JkiZ.png?dl=1" title="imagen" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/973709/181097201-4264f98e-d8fc-47b8-8457-0c53bd75a8c6.png" alt="imagen" data-base62-sha1="rH6H9ltmwKmlypnzDtkXr33JkiZ" width="690" height="377" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/c/2/c21a2b402e6b5effa09d581f3a15ab0dbf231191_2_690x377.png, https://aws1.discourse-cdn.com/github/original/3X/c/2/c21a2b402e6b5effa09d581f3a15ab0dbf231191.png 1.5x, https://aws1.discourse-cdn.com/github/original/3X/c/2/c21a2b402e6b5effa09d581f3a15ab0dbf231191.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/c/2/c21a2b402e6b5effa09d581f3a15ab0dbf231191_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">imagen</span><span class="informations">1034×565 31.9 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>What’s happening? :thinking: I did the same steps on  <a href="https://github.com/Tecnativa/doodba" rel="noopener nofollow ugc">https://github.com/Tecnativa/doodba</a> for example, and it shows the packages properly there.</p>

**Comment by clarkbw** [comment]
<p>I’m seeing it now on <a href="https://github.com/Tecnativa/docker-socket-proxy">https://github.com/Tecnativa/docker-socket-proxy</a> what changed?</p>

**Comment by yajo** [comment, marked as answer]
<p>Oh. Nothing, really, except some recent new pushes.<br>
Well, it seems github fixed it.</p>
